src/Proto.Actor/Future/Futures.cs

```diff
@@ -55,7 +55,8 @@ internal FutureProcess(ActorSystem system) : base(system)
 
             if (!absent) throw new ProcessNameExistException(name, pid);
 
-            Pid = pid.WithRequestId(1);
+            pid.RequestId = 1;
```

Comment:
seems to have some slight impact. not pushing GC too hard by creating new instances and ignoring the old